api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -602,25 +611,28 @@ def air_gap(
         self, volume: Optional[float] = None, height: Optional[float] = None
     ) -> InstrumentContext:
         """
-        Pull air into the pipette current tip at the current location
+        Draw air into the pipette's tip at the current location. See :ref:`air-gap`.
```

Comment:
Param description make clear this happens at the same _well_ but not at the same exact location (i.e., the pipette moves to get out of the well so it doesn't air-gap in a location full of liquid).
```suggestion
        Draw air into the pipette's tip at the current well. 
        See :ref:`air-gap`.
```